### ISSUE TITLE
Issue with results reproduction

### ISSUE THREAD

**Issue by vongkusolkit** [alexhe101/MSDDN#1]
Hello, 

Thank you so much for your great work and for providing the codes to the models!
I have some issues with replicating the results of your MSDDN model where the evaluation scores are much lower than the results in the paper.

These are the scores I'm getting:
_################## reference comparision #######################
metrics:    PSNR,     SSIM,   SAM,    ERGAS,  SCC,    Q
deep    [22.8659, 0.6162, 0.0975, 6.9769, 0.8274, 0.5052]
################## no reference comparision ####################
metrics:    D_lamda, D_s,    QNR
deep     [0.0968, 0.7544, 0.7544]_

Could you please provide some guidance on reproducing the model performance? 
If possible, could you also share the pretrained model you used for testing?

My email is vongkusolkit@wisc.edu

Thank you so much again for your work and I'm looking forward to hearing your reply.

**Comment by alexhe101** [comment]
Maybe you didn't use the correct dataset. The dataset I used in the paper can be found here: https://github.com/manman1995/Awaresome-pansharpening. Also, please pay attention to the settings of hyperparameters. The batch size is 4, the learning rate is usually 5e-4, and the optimizer used is Adam. As for gradient clipping, it is set to 4 for the WV2 dataset and 40 for WV3 and GF2.

I hope this can help you. If you can't be able to reproduce it successfully, I can provide you with pre-trained weights.

**Comment by vongkusolkit** [comment]
Thanks for your reply! 
I did use those datasets and hyperparameters.. would the only configurations I'll be changing be in option.yml and the file paths in demo_deep_methods.py? Or am I missing something else?

If you could provide me with the pre-trained weights, that would be greatly appreciated! Thank you so much.

**Comment by alexhe101** [comment]
I updated the model file and reduced the number of channels to obtain fewer parameters and more stable training. I have sent you the pre trained weights on the wv3 dataset via email.
